learn/sherif/main_test/src/test/java/frc/robot/subsystems/StabilizedDriveTrainTest.java

```diff
@@ -25,13 +27,23 @@ public void setup() {
         MockitoAnnotations.initMocks(this);
         when(f.getMotor(0)).thenReturn(front);
         when(f.getMotor(1)).thenReturn(back);
+
+        CANEncoder mocked_encoder = mock(CANEncoder.class);
+        when(mocked_encoder.getPosition()).thenReturn(10.0);
+        when(front.GetEncoder()).thenReturn(mocked_encoder);
+
         when(timer.get()).thenReturn(0.0);
     }
 
     private PIDLoop makePIDLoop() {
         return new PIDLoop(0.1, 0, 0.4);
     }
 
+    @Test
+    public void testHack() {
+        System.out.println(front.GetEncoder().getPosition());
```

Comment:
Let's not print from our tests.  Test output should be clean and just contain the success / failure from the test runner.